dotcom-rendering/src/components/Caption.tsx

```diff
@@ -57,23 +55,23 @@ const overlaidBottomPadding = (format: ArticleFormat) => {
 	`;
 };
 
-const overlaidStyles = (palette: Palette, format: ArticleFormat) => css`
+const overlaidStyles = (format: ArticleFormat) => css`
```

Comment:
Not a blocker, but if you want to follow emotion best practices, [dynamic styles would use the style prop](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles). Here we would set the `paddingBottom` based on format, and this `overlaidStyle` would no longer need to be a function..